### PR TITLE
Moved getKey method to utils and it gets invoked inside signKeymethod.

### PR DESCRIPTION
**Pull request by ARJUN-R34**
- The method to get the encrypted private key is moved to utils and it gets invoked inside **signKey** method.

- The method to sign the transactions **signKey** calls the **getKey** method to get the encrypted private key after validating the password.

- The encrypted private key is decrypted and the transaction is signed and sent to the blockchain.

- Updated method names.

- Token gets stored in the local storage when login function is called.

- Logout function is also implemented

### CODE REVIEW

File: src/utils/helper.js [hunk 1]
```diff
@@ -208,6 +208,32 @@ async function verifyPublicAddress({ address, authToken }) {
   return { response: data };
 }
 
+async function getKey({ password, authToken }) {
```
Comment (apoorvq) [inline src/utils/helper.js:211]: Change to pKey

File: src/index.js [hunk 2]
```diff
@@ -51,35 +50,21 @@ class PBTS {
...
 
-  async getKey({ password }) {
-    const { error: VALIDATE_PASSWORD_ERROR } = await validatePassword({ password, authToken: this.authToken });
-
-    if (VALIDATE_PASSWORD_ERROR) {
-      return { error: VALIDATE_PASSWORD_ERROR };
-    }
-
-    const { error: GET_ACCESS_TOKEN_ERROR, response: accessToken } = await getAccessToken({ params: { password }, authToken: this.authToken });
+  async signKey({
```
Comment (apoorvq) [inline src/index.js:53]: change to signTx

File: src/utils/helper.js (hunk 1 above)
Comment (ARJUN-R34) [reply src/utils/helper.js]: Done.

File: src/index.js (hunk 2 above)
Comment (ARJUN-R34) [reply src/index.js]: Changed to `signAndSendTx`

**Pull request merged**